### PR TITLE
Refactored and consolidated Auth config and fixed localhost setup

### PR DESCRIPTION
**Pull request by dmsakamoto**

### CODE REVIEW

File: server/config/environment/production.js [hunk 1]
```diff
@@ -3,5 +3,10 @@ module.exports = {
   postgres: {
     // The DATABASE_URL variable should already include the username, password, host, port, and database name
     uri: process.env.DATABASE_URL
+  },
+  facebook: {
+    id: process.env.FACEBOOK_ID,
+    secret: process.env.FACEBOOK_SECRET,
+    callback: process.env.IP + '/Server/users/facebook/callback'
```
Comment (kevhuang) [inline server/config/environment/production.js]: @dmsakamoto / @stevenshuhyow should this callback property be defined as an environment variable just like id and secret?

Also, is this the correct path? Should it be /Server/auth/facebook/callback? Or should it be auth/facebook/callback? See lines 28 and 29 in routes.js for context.

**Comment by kevhuang** [comment]
@dmsakamoto you forgot to remove thirdPartyServices.js from the repo.

File: server/config/environment/production.js (hunk 1 above)
Comment (dmsakamoto) [inline server/config/environment/production.js]: I just followed the old code from @stevenshuhyow. @stevenshuhyow please advise

Comment (ekong2) [inline server/config/environment/production.js]: it is auth/facebook/callback, just like in test

**Comment by ekong2** [comment]
:+1:

**Comment by stevenshuhyow** [comment]
:+1:

**Pull request merged**